docs/DeveloperGuide.md

```diff
@@ -1,34 +1,561 @@
-# Developer Guide
+# Developer Guide for ra.VI
 
-## Design & implementation
+## Table of Contents
 
-{Describe the design and implementation of the product. Use UML diagrams and short code snippets where applicable.}
+## Setting up
+The following section describes how to set up ra.VI on your own computer.
 
+### Software Prerequisites
```

Comment:
The glow effect on the chosen webpage style might make it hard to tell level 2 and level 3 headings apart. Numbering the sub-sections should help with this greatly.
![image](https://user-images.githubusercontent.com/69422560/97667142-2f049200-1aba-11eb-83c8-1373058ae46b.png)